custom-markdownlint-rules/README.md

```diff
@@ -0,0 +1,13 @@
+# Custom Markdownlint Rules
+
+## Directory Link has Trailing Slash
+
+Markdown links that point to directories must end in `/`.
+
+## Valid Anchor Reference
+
+Markdown links with anchor tag references must link to a real heading in the referenced file.
+
+## Valid File Reference
+
+Markdown links to files and directories must resolve to a file or directory in the project.
```

Comment:
Tests for all rules are added in later commits. The tests provide more examples for each rule